app/exec/build-tasks-upload.ts

```diff
@@ -25,33 +27,55 @@ export class BuildTaskUpload extends cmdm.TfCommand {
     public exec(args: string[], options: cm.IOptions): any {
         var taskPath = args[0] || options['taskPath'];
         this.checkRequiredParameter(taskPath, 'taskPath', 'path to the task folder');
```

Comment:
you need to also validate this is a valid directory.  ok to use fs sync methods.  there's also shelljs.test('-d' ...